src/patternfly/components/Breadcrumb/breadcrumb.scss

```diff
@@ -0,0 +1,125 @@
+@import "../../patternfly-utilities";
+
+.pf-c-breadcrumb {
+  --pf-c-breadcrumb--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-breadcrumb--PaddingRight: var(--pf-global--spacer--xl);
+  --pf-c-breadcrumb--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-breadcrumb--PaddingLeft: var(--pf-global--spacer--xl);
+
+  // Breadcrumb Item
+  --pf-c-breadcrumb__item--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-breadcrumb__item--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-breadcrumb__item--LineHeight: var(--pf-global--LineHeight--sm);
+  --pf-c-breadcrumb__item--Color: var(--pf-global--Color--300);
+  --pf-c-breadcrumb__item--hover--Color: var(--pf-global--Color--300);
+
+  // Breadcrumb Link
+  --pf-c-breadcrumb__link--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-breadcrumb__link--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-breadcrumb__link--LineHeight: var(--pf-global--LineHeight--sm);
+  --pf-c-breadcrumb__link--Color: var(--pf-global--link--Color);
+  --pf-c-breadcrumb__link--hover--Color: var(--pf-global--link--Color);
+  --pf-c-breadcrumb__link--hover--TextDecoration: var(--pf-global--link--TextDecoration--hover);
+
+  // Breadcrumb Divider
+  --pf-c-breadcrumb__divider--MarginRight: var(--pf-global--spacer--sm);
+  --pf-c-breadcrumb__divider--MarginLeft: var(--pf-global--spacer--sm);
+  --pf-c-breadcrumb__divider--FontFamily: "Font Awesome 5 Free";
```

Comment:
Should we add a global var for the font awesome `font-family` so we can reuse that in other components that apply the fa styles directly to a pseudo element?

Reply:
I concur :)